readthedocs/config/tests/test_yaml_loader.py

```diff
@@ -12,9 +12,13 @@ def test_yaml_load_safely():
     content = textwrap.dedent('''
     int: 3
     float: !!float 3
-    unknown: !!python/name:unknown_funtion
+    name: !!python/name:python_function
```

Comment:
Maybe a test for `!!python/name:module.other.function` is useful as well.

Reply:
Done.